vshard-scm-1.rockspec

```diff
@@ -10,7 +10,12 @@ description = {
     license  = 'BSD',
 }
 dependencies = {
-    'tarantool >= 1.9.0';
+    'lua >= 5.1';
+}
+external_dependencies = {
+    TARANTOOL = {
+        header = 'tarantool/module.h';
```

Comment:
But vshard does not depend in tarantool headers.

Reply:
Hm. We can remove that, yep. I added this to match tarantool/rocks specs.